src/main/resources/documentation/domain-model-overview.md

```diff
@@ -0,0 +1,257 @@
+#Contract Entities
+####AuditableEntity

```

Comment:
I vote for not documenting the abstract classes.  Users probably don't care about them, and programmers can probably figure them out by reading the code.